app/dash_app.py

```diff
@@ -40,6 +42,15 @@
 )
 args = parser.parse_args()
 
+logging.basicConfig(
```

Comment:
A couple comments on logging:

* I would not log to a file within the application logic, as this may make it difficult to capture logging output in cloudwatch. 
* It would be useful to make logging level configurable via CLI arguments and/or an environment variable, with a default level set as a fallback.

Reply:
Resolved with https://github.com/Volcano-Risk-Reduction-in-Canada/Volcanic-Interpretation-Workbench/pull/68/commits/14312e8a79a25f57a07692e919fbadf83d387429